.circleci/config.yml

```diff
@@ -129,15 +159,60 @@ jobs:
           paths:
             - nucypher-*
 
+  pip_install:
+    <<: *python_36_base
+    steps:
+      - checkout
+      - run:
+          name: Install Python Dependencies with Pip
+          command: |
+            pip3 install --user .
+      - run:
+          name: Install Python Development Dependencies with Pip
+          command: |
+            pip3 install --user -e .[testing]
```

Comment:
When we are testing, do we even need `-e`?

Reply:
True, we don't need it.

Reply:
:heavy_check_mark: